src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/media/fixtures.js

```diff
@@ -0,0 +1,101 @@
+export const CPSMediaBlock = {
+  id: 'p07jl3lv',
+  subType: 'clip',
+  format: 'video',
+  title: 'Why dis pikin get fish skin',
+  synopses: {
+    short: 'Short Synopsis',
+    long: 'Long Synopsis',
+    medium: 'Medium Synopsis',
+  },
+  imageUrl: 'ichef.bbci.co.uk/images/ic/$recipe/p07jl8g4.jpg',
+  embedding: true,
+  advertising: true,
+  caption: 'Warning: Dis video fit make you feel one kain!',
+  versions: [
+    {
+      versionId: 'p07jl3lx',
+      types: ['Original'],
+      duration: 101,
+      durationISO8601: 'PT1M41S',
+      warnings: {
+        long: 'This is a long warning',
+        short: 'This is a short warning',
+      },
+      availableTerritories: {
+        uk: true,
+        nonUk: true,
+      },
+      availableFrom: 1564740166,
+    },
+  ],
+  imageCopyright: 'BBC',
+  type: 'media',
+};
+
+export const optimoVideoBlock = {
+  type: 'video',
```

Comment:
![image](https://user-images.githubusercontent.com/11341355/67307012-bd315a00-f4ef-11e9-8950-17be3bcffbd7.png)
https://www.test.bbc.co.uk/news/articles/c3wmq4d1y3wo.json doesnt have `type` and `locator` on the same level

Reply:
ooh, nice catch, fixed